code/modules/holiday/new_year.dm

```diff
@@ -0,0 +1,111 @@
+//~*~*~*~*SPARKLER*~*~*~*~*~*~

```

Comment:
This file doesn't belong here imo. I'd suggest `code/game/objects/items/grenades/festive.dm`

Reply:
I have implemented your suggestion, they are now under grenades. Probably a good choice if the number of fireworks should expand in the future.